SU-HSDO/suhumsci
# feat: Lando integration for local development

# READY FOR REVIEW

## Summary
Updates the old lando setup with a more module/extendable approach for multisites.
Updates setup documentation

## Need Review By (Date)
TBD

## Urgency
high

## Steps to Test
1. Review the README documentation in `/lando/README.md` and follow the setup steps. Note: if you are using an ARM-based Mac you will need to follow additional setup steps as there are known compatibility issues with lando/docker and the M1/M2 Macs.
2. Verify you can sync and run multiple site's at once
